patroni/api.py

```diff
@@ -230,22 +229,30 @@ def do_POST_restart(self):
                     status_code = 400
             else:
                 request['postmaster_start_time'] = self.server.patroni.ha.state_handler.postmaster_start_time()
-                if self.server.patroni.ha.schedule_future_restart(request):
-                    data = "Restart scheduled"
-                    status_code = 202
-                else:
-                    data = "Another restart is already scheduled"
-                    status_code = 409
+                try:
+                    if self.server.patroni.ha.schedule_future_restart(request):
+                        data = "Restart scheduled"
+                        status_code = 202
+                    else:
+                        data = "Another restart is already scheduled"
+                        status_code = 409
+                except Exception as e:
```

Comment:
Actually it doesn't make sense to catch exception here, because`touch_member` is already catching everything related...
Sorry, my bad...